src/script/components/message.mjs

```diff
@@ -0,0 +1,5 @@
+const loader = document.querySelector(".loader");
```

Comment:
This is a good idea. You could improve it and make it an alert that shows and dissappears after a given time. It can replace the default alert.